opsmngr/live_data_migration.go

```diff
@@ -24,25 +24,30 @@ import (
 
 const liveMigrationBasePath = "api/public/v1.0/orgs/%s/liveExport/migrationLink"
 
-// LiveMigrationService is an interface for interfacing with the Live Migration
+// LiveDataMigrationService is an interface for interfacing with the Live Migration
 // endpoints of the MongoDB Ops Manager API.
 //
 // See more: https://docs.opsmanager.mongodb.com/current/reference/api/cloud-migration/
-type LiveMigrationService interface {
-	Create(context.Context, string, *atlas.LinkToken) (*atlas.LiveMigration, *Response, error)
-	Delete(context.Context, string) (*Response, error)
-	Get(context.Context, string) (*atlas.LiveMigration, *Response, error)
+type LiveDataMigrationService interface {
+	ConnectOrganizations(context.Context, string, *atlas.LinkToken) (*ConnectionStatus, *Response, error)
+	DeleteConnection(context.Context, string) (*Response, error)
+	ConnectionStatus(context.Context, string) (*ConnectionStatus, *Response, error)
 }
 
-// LiveMigrationOp provides an implementation of the LiveMigrationService interface.
-type LiveMigrationServiceOp service
+// ConnectionStatus represents the response of LiveDataMigrationService.ConnectOrganizations and LiveDataMigrationService.ConnectionStatus
```

Comment:
`golangci-lint` is showing a missing `.` at the end of the line here (also cool interface in github showing that!)

Reply:
This issue has been fixed in the last commit